sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -622,6 +643,88 @@ def _build_item(self, item):
         return item
 
 
+class FilteredBlob(FilterBlobItem):
+    """Blob info from a Filter Blobs API call.
+
+    :ivar name: Blob name
+    :type name: str
+    :ivar container_name: Container name.
+    :type container_name: str
+    :ivar tag_value: tag value filtered by the expression.
+    :type tag_value: str
+    """
+    def __init__(self, **kwargs):  # pylint:disable=useless-super-delegation
+        super(FilteredBlob, self).__init__(**kwargs)
```

Comment:
why do you need this super delegation? It's unnecessary

Reply:
We don't want to expose the generated code to customer right?